lib/msal-node/src/crypto/GuidGenerator.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { v4 as uuidv4 } from 'uuid';
+
+export class GuidGenerator {
```

Comment:
Should this class be in common? we use guids a bit in browser lib

Reply:
well it can be, if @pkanher617 agrees. I can move this to common and have both libraries use the same.